library/Ivoz/Provider/Domain/Model/Friend/FriendDto.php

```diff
@@ -2,15 +2,92 @@
 
 namespace Ivoz\Provider\Domain\Model\Friend;
 
+use Ivoz\Api\Core\Annotation\AttributeDefinition;
+use Ivoz\Kam\Domain\Model\UsersLocation\Status;
+
 class FriendDto extends FriendDtoAbstract
 {
+    const CONTEXT_STATUS = 'status';
+
+    /**
+     * @var Status[]
+     * @AttributeDefinition(
+     *     type="array",
+     *     class="Ivoz\Kam\Domain\Model\UsersLocation\Status",
+     *     description="Terminal status"
+     * )
+     */
+    protected $status = [];
+
+    /**
+     * @var string
+     * @AttributeDefinition(
+     *     type="string",
+     *     description="Terminal domain"
```

Comment:
description="Registration domain" is a description that you can use for all these endpoints